### ISSUE TITLE
QForm, QFile, QUploader not supported? 

### ISSUE THREAD

**Issue by mjmare** [elimintz/justpy#81]
The QUploader is very feature rich so it would be nice to have it available. (I know you have a non Quasar file upload example).

QForm is needed for 'internal' validation of QInputs (https://quasar.dev/vue-components/form). OTOH you seem to support QInput 'rules' (https://justpy.io/quasar_tutorial/QInput/#input-validation).
Maybe needed for QUploader as well.

BTW are the components on https://justpy.io/quasar_tutorial/not_working/ fundamentally incompatible or do they just need more work?

**Comment by elimintz** [comment]
I have to look again at QUploader. I made a few changes to support QInput rules so maybe that will help.

You ask a good question. I don't know for sure about the compatibility of the not working components. The guys in the Quasar community are very helpful so I'll ask them soon about this.

**Comment by mjmare** [comment]
It is unavoidable that some Quasar components or features are not supported for the foreseeable time. It would be helpful to expand the 'not supported' page so new users will not have to guess. 
As you have shown some non working features can worked around. So maybe add these workarounds to the 'not supported' page.

**Comment by elimintz** [comment]
Yes, that is a good idea. Any suggestions on how to automate testing of the many Quasar components and their options would also be appreciated.

**Comment by mjmare** [comment]
OMG javascript testing

**Comment by elimintz** [comment]
No, just testing on the Python side to make sure that the components are defined correctly. Like catching the default_expand_all bug that you found. The question is how to automate this since the 'truth' comes from the component working properly in the browser.
